install/installer/pkg/components/server/pdb.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package server
+
+import (
+	"github.com/gitpod-io/gitpod/installer/pkg/common"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func pdb(ctx *common.RenderContext) ([]runtime.Object, error) {
+	return []runtime.Object{
+		common.PodDisruptionBudget(ctx, Component, 1, &metav1.LabelSelector{
```

Comment:
how does it work? As far as I understand we have only one instance of server in dedicated? With max unavailable 1 it would mean downtime? sorry not familiar with PodDisruptionBudget

Reply:
> As far as I understand we have only one instance of server in dedicated?

We have two for the majority of the meta cluster components